posts/2025-03-11_code_collaborati.../code_collaboration_in__ne_st.qmd

```diff
@@ -0,0 +1,78 @@
+date: "2025-03-11"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [TLG, Community]
+# feel free to change the image
+image: "logo.svg"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2025-03-11_code_collaborati..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+In the ever-evolving field of clinical data analysis, innovation and collaboration are key. Today, we are pleased to share an initial, yet important milestone as part of a collaboration effort between Roche and Clinical & Statistical Programming, IDAR, J&J Innovative Medicine (further referred to as C&SP IDAR) within the NEST framework.
+
+### Table creation in clinical reporting in pharma
+
+In clinical trials, the presentation of data is as crucial as the data itself. Clear, accurate, and standardized tables ensure that the results are interpreted correctly and that regulatory bodies can easily review and approve submissions. 
+
+For an overview of creation of tables within end-to-end clinical reporting in pharma, both [Pharmaverse TLGs Overview](https://pharmaverse.org/e2eclinical/tlg/) and the effort from [R Consortium](https://rconsortium.github.io/rtrs-wg/) are a useful read to compare examples from several packages.
+
+### Code collaboration
+
+Moving forward at C&SP IDAR, we are transitioning towards adopting R as our preferred analytical tool, and we were thrilled with the capabilities within the NEST framework ([`{rtables}`](https://insightsengineering.github.io/rtables/){target="_blank"} [`{tern}`](https://insightsengineering.github.io/tern/){target="_blank"} and [`{teal}`](https://insightsengineering.github.io/teal/){target="_blank"}) designed by Roche. 
+
+As of today, we are building our new internal standard table shells using the open source packages from this NEST framework.
+
+The `{rtables}` package is designed to simplify and standardize the process of creating tables for clinical trial reports. It offers a flexible and robust framework for generating high-quality, reproducible tables that adhere to industry standards. 
+
+While the framework is robust in terms of type of analysis that can be performed, a key requirement for `C&SP IDAR` was not met. Our table output format requirement is the usage of `rtf` with `non-monospace` fonts.
+
+As such, the initial scope of our collaboration was focusing on this **Output Format Customization** requirement. 
+
+We are proud to share the first result of the collaboration effort! With the current  [CRAN](https://cran.r-project.org/web/packages/rtables/index.html){target="_blank"} release of `{rtables}`, we can achieve outputs to be created in our desired output format. See the [NEWS updates](https://cran.r-project.org/web/packages/rtables/news/news.html){target="_blank"} (rtables 0.6.8) to find out more details on the new features.
+
```

Comment:
MIght be nice here to break up the text and post with an example output showcasing the non-monospace fonts in action?